python/pyspark/sql/udf.py

```diff
@@ -253,6 +250,7 @@ def __init__(
     def returnType(self) -> DataType:
         # This makes sure this is called after SparkContext is initialized.
         # ``_parse_datatype_string`` accesses to JVM for parsing a DDL formatted string.
+        # TODO: PythonEvalType.SQL_BATCHED_UDF
```

Comment:
@xinrong-meng just curious what is the TODO item here?

Reply:
This is why we should have ID'ed todo .. like `TODO(SPARK-XXXXX): ...`